source/PhysicalQuantity/Time.php

```diff
@@ -71,6 +71,26 @@ protected static function initialize()
         $newUnit->addAlias('gregorian years');
         static::addUnit($newUnit);
 
+		// Decade
+		$newUnit = UnitOfMeasure::linearUnitFactory('dc', 315569520);
+		$newUnit->addAlias('decade');
+		$newUnit->addAlias('decades');
+		static::addUnit($newUnit);
+
+		// Century
+		$newUnit = UnitOfMeasure::linearUnitFactory('cnt', 3155695200);
+		$newUnit->addAlias('century');
+		$newUnit->addAlias('centuries');
+		static::addUnit($newUnit);
+
+		// Millennium
+		$newUnit = UnitOfMeasure::linearUnitFactory('mln', 31556952000);
```

Comment:
I think the only accepted spelling is "millennium", so the single-n versions should be dropped.  Also, plural of millennium is "millennia"